routes/settings.js

```diff
@@ -101,7 +101,10 @@ module.exports.postSettings = async (request, response, next) => {
     try {
       await createClient({
         space: spaceId,
-        accessToken: deliveryToken
+        accessToken: deliveryToken,
+        // Environment variable is used here to enable testing this app internally at Contentful.
+        // You can just omit the host since it defaults to 'cdn.contentful.com'
+        host: process.env.CONTENTFUL_DELIVERY_API_HOST
```

Comment:
we can just say `CONTENTFUL_API_HOST` here and always pass it for both `delivery` and `preview`